cookbooks/Berksfile

```diff
@@ -19,7 +19,9 @@ Chef::Config.chef_repo_path = Chef::Config.find_chef_repo_path(__dir__)
 Chef::LocalMode.with_server_connectivity do
   require 'chef/server_api'
   cookbooks = Chef::ServerAPI.new(Chef::Config[:chef_server_url]).get('/cookbooks')
-  cookbooks.each_key do |path|
+  cookbooks.keys.select do |path|
+    path.start_with? 'cdo-'
```

Comment:
This change is necessary for us to avoid attempting to treat the `vendor` directory produced by `bundle install` as a cookbook. It does commit us to always prefixing all of our cookbooks with `cdo-`, but as [that is the case](https://github.com/code-dot-org/code-dot-org/tree/staging/cookbooks) and has been for quite some time, I feel comfortable with that.